site/class-1-bar/data.tsv

```diff
@@ -0,0 +1,14 @@
+date	steps
```

Comment:
You are uploading your files directly in `site/class-1-bar/` but they should go in `site/class-1-bar/sophiemei`.

You should first add a readme in `site/class-1-bar/sophiemei/readme.md`, and then upload your files to `site/class-1-bar/sophiemei/`.

Finally, you can click the trash icon on the top-right of files to remove them here on the GitHub interface.